src/main/java/com/jfeatures/msg/codegen/GenerateDTO.java

```diff
@@ -45,9 +45,11 @@ public static JavaFile dtoFromSqlAndDdl(String sql, Map<String, String> ddlPerTa
         return javaFile;
     }
 
+
     private static TypeSpec getDto(String businessPurposeOfSQL, Map<TableColumn, ColumnDefinition> columnNameToTypeMapping, List<FieldSpec> fieldSpecList) {
 
         TypeSpec dto;
+        //TODO: why 255?
```

Comment:
Java has a limit on the number of parameters a method can have: https://stackoverflow.com/questions/1309230/java-compile-error-parameter-x-is-exceeding-the-limit-of-255-words

Reply:
Removed and add some doc about 255